crates/bevy_color/src/color.rs

```diff
@@ -383,6 +381,64 @@ impl Alpha for Color {
     }
 }
 
+/// All color channels are scaled directly,
+/// but alpha is unchanged.
+///
+/// Values are not clamped.
+impl Mul<f32> for Color {
+    type Output = Self;
+
+    fn mul(self, rhs: f32) -> Self::Output {
+        match self {
+            Color::Srgba(x) => (x * rhs).into(),
+            Color::LinearRgba(x) => (x * rhs).into(),
+            Color::Hsla(x) => (x * rhs).into(),
+            Color::Hsva(x) => (x * rhs).into(),
+            Color::Hwba(x) => (x * rhs).into(),
+            Color::Laba(x) => (x * rhs).into(),
+            Color::Lcha(x) => (x * rhs).into(),
+            Color::Oklaba(x) => (x * rhs).into(),
+            Color::Oklcha(x) => (x * rhs).into(),
+            Color::Xyza(x) => (x * rhs).into(),
+        }
+    }
+}
+
+/// All color channels are scaled directly,
+/// but alpha is unchanged.
+///
+/// Values are not clamped.
+impl Mul<Color> for f32 {
+    type Output = Color;
+
+    fn mul(self, rhs: Color) -> Self::Output {
+        rhs * self
+    }
+}
+
+/// All color channels are scaled directly,
+/// but alpha is unchanged.
+///
+/// Values are not clamped.
+impl Div<f32> for Color {
+    type Output = Self;
+
+    fn div(self, rhs: f32) -> Self::Output {
+        match self {
+            Color::Srgba(x) => (x / rhs).into(),
+            Color::LinearRgba(x) => (x / rhs).into(),
+            Color::Hsla(x) => (x / rhs).into(),
+            Color::Hsva(x) => (x / rhs).into(),
+            Color::Hwba(x) => (x / rhs).into(),
+            Color::Laba(x) => (x / rhs).into(),
+            Color::Lcha(x) => (x / rhs).into(),
+            Color::Oklaba(x) => (x / rhs).into(),
+            Color::Oklcha(x) => (x / rhs).into(),
+            Color::Xyza(x) => (x / rhs).into(),
+        }
+    }
+}
+
```

Comment:
```suggestion
```

Maths operations on the `Color` type are _very_ poorly defined to the point of uselessness. It would always be preferable (and more performant!) to require users to transform `Color` into a concrete space first, operate, and then wrap in `Color` again.